app/services/check_answers/section_lister_service.rb

```diff
@@ -80,16 +80,11 @@ def build_field(field_data, model, table_label, index: nil)
       return if field_data[:skip_unless].present? && !model.send(field_data[:skip_unless])
       return if field_data[:skip_if].present? && model.send(field_data[:skip_if])
 
-      label = if index
-                index.zero? ? "#{field_data.fetch(:attribute)}_first" : "#{field_data.fetch(:attribute)}_subsequent"
-              else
-                field_data.fetch(:attribute)
-              end
       addendum = "_partner" if @check.partner && field_data[:partner_dependant_wording]
 
       disputed = field_data[:disputed_if].present? && @check.smod_applicable? && model.send(field_data.fetch(:disputed_if))
 
-      Field.new(label: "#{table_label}_fields.#{label}#{addendum}",
+      Field.new(label: "#{table_label}_fields.#{field_data.fetch(:attribute)}#{addendum}",
```

Comment:
When the `field_data[:many]` attribute was set (only happened for bank accounts), we could have an arbitrary number of rows within a table (for all other "add another" patterns, we would add arbitrary numbers of tables, not rows within tables).  That attribute would trigger the `build_many_fields` method to run, which would then call `build_field` passing in an `index`. The old code would detect the presence of that index and if it was there, use `_first` and `_subsequent` to construct the labels (so the first one would be "Money in bank accounts" and the second onwards would be "Additional bank account X"). Now we don't need any of that, because the format for all rows is "Money in bank account X".